src/rollup/display-conditions/url-contains/url-checker.js

```diff
@@ -0,0 +1,55 @@
+/** @module pathfora/display-conditions/url-contains/url-checker */
+
+// utils
+import escapeURI from '../../utils/url/escape-uri';
+
+// display conditions
+import parseQuery from './parse-query';
+import phraseChecker from './phrase-checker';
+
+/**
+ * Evaluate if the current URL matches the rules defined
+ * by the urlContains display condition
+ *
+ * @exports urlChecker
+ * @params {array} phrases
+ * @returns {boolean}
+ */
+export default function urlChecker (phrases) {
+  var url = escapeURI(window.location.href, { keepEscaped: true }),
+      simpleurl = window.location.hostname + window.location.pathname,
+      queries = parseQuery(url),
+      valid, excludeValid = false,
+      matchCt, excludeCt = 0;
+
+  if (!(phrases instanceof Array)) {
+    phrases = Object.keys(phrases).map(function (key) {
+      return phrases[key];
+    });
+  }
+
+  // array of urlContains params is an or list, so if any are true evaluate valid to true
```

Comment:
super nit but might be worth capitalizing `or` here so it doesn't read like `params is an or` ... an or what? :)